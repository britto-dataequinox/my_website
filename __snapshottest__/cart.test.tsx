import Cart from '@/pages/cart';
import React from 'react';
import renderer from 'react-test-renderer';

jest.mock('next/router', () => ({
  useRouter() {
    return {
      route: '/',
      pathName: '',
      query: '',
      asPath: '/',
      push: jest.fn(),
    };
  },
}));

test('cart snap', () => {
  const tree = renderer.create(<Cart />).toJSON();
  expect(tree).toMatchSnapshot();
});
