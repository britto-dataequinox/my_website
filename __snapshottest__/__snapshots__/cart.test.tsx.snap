// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cart snap 1`] = `
[
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-rjnxjj-MuiPaper-root-MuiAppBar-root"
  >
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
    >
      <div
        className="MuiTypography-root MuiTypography-h6 css-1bn9r4q-MuiTypography-root"
        onClick={[Function]}
      >
        Online Store
      </div>
    </div>
  </header>,
  <div
    className="MuiBox-root css-iri6d5"
  >
    <div
      className="MuiBox-root css-ww7h5q"
    >
      Cart work not getting started
    </div>
  </div>,
]
`;
